### ISSUE TITLE
Test failure: JIT/SIMD/MinMax_ro/MinMax_ro.sh

### ISSUE THREAD

**Issue by v-haren** [dotnet/runtime#38799]
failed in job: [runtime-coreclr outerloop 20200704.3 ](https://dev.azure.com/dnceng/public/_build/results?buildId=716701&view=ms.vss-test-web.build-test-results-tab&runId=22210646&resultId=109769&paneView=debug)

Error message
~~~
/root/helix/work/workitem/JIT/SIMD/MinMax_ro/MinMax_ro.sh: line 160: 957 Trace/breakpoint trap __Command=$_DebuggerFullPath "$CORE_ROOT/crossgen" /Platform_Assemblies_Paths $CORE_ROOT:$PWD /in MinMax_ro.org /out MinMax_ro.dll
Managed assembly not found: No such file or directory

Return code: 1
Raw output file: /root/helix/work/workitem/JIT/SIMD/Reports/JIT.SIMD/MinMax_ro/MinMax_ro.output.txt
Raw output:
BEGIN EXECUTION
in takeLock

in ReleaseLock
in takeLock
/root/helix/work/correlation/corerun MinMax_ro.dll ''
in ReleaseLock
Expected: 100
Actual: 255
END EXECUTION - FAILED
Test Harness Exitcode is : 1
To run the test:
> set CORE_ROOT=/root/helix/work/correlation
> /root/helix/work/workitem/JIT/SIMD/MinMax_ro/MinMax_ro.sh
Expected: True
Actual: False


Stack trace
   at JIT_SIMD._MinMax_ro_MinMax_ro_._MinMax_ro_MinMax_ro_sh() in /__w/1/s/artifacts/tests/coreclr/Linux.arm64.Checked/TestWrappers/JIT.SIMD/JIT.SIMD.XUnitWrapper.cs:line 3535
~~~

category:correctness
theme:vector-codegen
skill-level:intermediate
cost:medium

**Comment by JulieLeeMSFT** [comment]
CC @erozenfeld

**Comment by BruceForstall** [comment]
Same test failed in https://github.com/dotnet/runtime/issues/38496

This failure is `R2R Linux arm64 Checked @ (Ubuntu.1804.Arm64.Open)`

**Comment by erozenfeld** [comment]
#38496 was fixed by #38591. This is the same test but very different failure: `Managed assembly not found: No such file or directory`

**Comment by BruceForstall** [comment]
Looks like this fails in all the Linux arm64 R2R runs:

https://dev.azure.com/dnceng/public/_build/results?buildId=716993&view=ms.vss-test-web.build-test-results-tab&runId=22219450&resultId=110107&paneView=debug

https://dev.azure.com/dnceng/public/_build/results?buildId=716582&view=ms.vss-test-web.build-test-results-tab&runId=22207570&resultId=110525&paneView=debug
